flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -439,7 +439,7 @@ public CheckpointingMode getCheckpointingMode() {
 	 * from operations on {@link org.apache.flink.streaming.api.datastream.KeyedStream}) is maintained
 	 * (heap, managed memory, externally), and where state snapshots/checkpoints are stored, both for
 	 * the key/value state, and for checkpointed functions (implementing the interface
-	 * {@link org.apache.flink.streaming.api.checkpoint.Checkpointed}).
+	 * {@link org.apache.flink.streaming.api.checkpoint.CheckpointedFunction}).
```

Comment:
> (for example implementing the interface {@link org.apache.flink.streaming.api.checkpoint.ListCheckpointed})

`CheckpointedFunction` is also deprecated

Reply:
Thanks @pnowojski for review. I am appreciate it! Hmm, I am not sure if it exists maven plugins tools for checking the same thing. I will updated the code based on your suggestions. Then, watch the test what will happen. Why did I put it in ```manual``` package, because I found a class named ```CheckForbiddenMethodsUsage``` might did the same stuff. So, I also did it like this. Do you have any good idea where it should put ?

Reply:
I have changed it to ```ListCheckpointed``` class instead.